website/docs/cloud-docs/policy-enforcement/import-reference/index.mdx

```diff
@@ -0,0 +1,25 @@
+---
+page_title: Sentinel import function reference
+description: Use the Sentinel import function to configure policy behaviors in custom Sentinel policies. 
+---
+# `import` function reference overview
+
+This topic provides an overview of the Sentinel `import` functions you can use in custom Sentinel policies. Refer to [Define custom Sentinel policies](/terraform/cloud-docs/policy-enforcement/define-policies/custom-sentinel) for additional information about how to use the functions.
+
+## Functions for Terraform
+You can include Sentinel `import` functions that enable a policy to access reusable libraries, external data, and functions. Refer to the [Sentinel imports documentation](/sentinel/docs/language/imports) for more details.
+
+HCP Terraform provides the following `import` functions to define policy rules for the plan, configuration, state, and run associated with a policy check.
+
+- [`tfplan`](/terraform/cloud-docs/policy-enforcement/import-reference/tfplan) : Provides access to a Terraform plan, which is the file created when you run the `terraform plan` command. This function is deprecated. Use `tfplanv/2` instead.
```

Comment:
```suggestion
HCP Terraform provides the following importable libraries to define policy rules for the plan, configuration, state, and run associated with a policy check.

- [`tfplan`](/terraform/cloud-docs/policy-enforcement/import-reference/tfplan) : Provides access to a Terraform plan, which is the file created when you run the `terraform plan` command. This library is deprecated. Use `tfplanv/2` instead.
```
Again, I thought this was the correct terminology, ignore me if I'm wrong! :)

Otherwise, same comment applies to the rest of the list.

Reply:
No, thank you for addressing this. I wasn't sure what the correct terminology was. I just know that we shouldn't refer to them as "imports", like we're bringing cheese over from Italy.